frontend/src/components/ManagePermissionsTable/index.vue

```diff
@@ -105,8 +105,11 @@ const props = defineProps<{
     addNotifications: (newNotifications: PermissionNotificationType[]) => void;
 }>();
 
-const isAppAdminTable = props.tableType === ManagePermissionsTableEnum.AppAdmin;
+const isAppAdminTable =
```

Comment:
Hi Jazz, hope it is not too bother to you.
Since now the enum is refactored to be `FamAppAdmin`, could you also refactor this const from `isApplicationAdminTable` to `isFamApplicationAdminTable` to be consistent?

Also on line 117 for code `const appAdminQuery = useQuery(`,  to be `const famAppAdminQuery =...`.
 Thank you!